src/main/java/org/spongepowered/api/data/key/Keys.java

```diff
@@ -76,52 +95,184 @@
+     * if there is no other Player around who affects spawning.</p>
+     *
      * @see AffectsSpawningData#affectsSpawning()
      */
     public static final Key<Value<Boolean>> AFFECTS_SPAWNING = KeyFactory.fake("AFFECTS_SPAWNING");
 
+    /**
+     * Represents the {@link Key} for the age of any {@link Ageable} creature
+     * in ticks.
+     *
+     * @see AgeableData#age()
+     */
     public static final Key<MutableBoundedValue<Integer>> AGE = KeyFactory.fake("AGE");
 
+    /**
+     * Represents the {@link Key} for whether an {@link Agent}s AI is enabled.
+     *
+     * @see AgentData#aiEnabled()
+     */
     public static final Key<Value<Boolean>> AI_ENABLED = KeyFactory.fake("AI_ENABLED");
 
+    /**
+     * Represents the {@link Key} for how angry an {@link Entity} is. This
+     * applies mostly to {@link ZombiePigman}.
+     *
+     * <p>Unlike {@link #ANGRY}, the agressiveness represented by this key may
+     * fade over time and the entity will become peaceful again once its anger
+     * reaches its minimum.</p>
+     *
+     * @see AngerableData#angerLevel()
+     */
     public static final Key<MutableBoundedValue<Integer>> ANGER = KeyFactory.fake("ANGER");
 
+    /**
+     * Represents the {@link Key} for whether an {@link Entity} is currently
+     * aggressive. This mostly applies to wolves.
+     *
+     * @see AggressiveData#aggressive()
+     */
     public static final Key<Value<Boolean>> ANGRY = KeyFactory.fake("ANGRY");
 
+    /**
+     * Represents the {@link Key} for the age (in ticks) of an
+     * {@link AreaEffectCloud} created by a lingering potion.
+     *
+     * @see AreaEffectCloudData#age()
+     */
     public static final Key<MutableBoundedValue<Integer>> AREA_EFFECT_CLOUD_AGE = KeyFactory.fake("AREA_EFFECT_CLOUD_AGE");
 
+    /**
+     * Represents the {@link Key} for the color of an {@link AreaEffectCloud}
+     * created by a lingering potion.
+     *
+     * @see AreaEffectCloudData#color()
+     */
     public static final Key<Value<Color>> AREA_EFFECT_CLOUD_COLOR = KeyFactory.fake("AREA_EFFECT_CLOUD_COLOR");
 
+    /**
+     * Represents the {@link Key} for the maximum age (in ticks) of an
+     * {@link AreaEffectCloud} created by a lingering potion.
+     *
+     * @see AreaEffectCloudData#duration()
+     */
     public static final Key<MutableBoundedValue<Integer>> AREA_EFFECT_CLOUD_DURATION = KeyFactory.fake("AREA_EFFECT_CLOUD_DURATION");
 
-    public static final Key<MutableBoundedValue<Integer>> AREA_EFFECT_CLOUD_DURATION_ON_USE = KeyFactory.fake("AREA_EFFECT_CLOUD_DURATION_ON_USE");
+    /**
+     * Represents the {@link Key} for the amount of ticks the duration of an
+     * {@link AreaEffectCloud} is increased or reduced when it applies its
+     * effect.
+     *
+     * @see AreaEffectCloudData#durationOnUse()
+     */
+    public static final Key<MutableBoundedValue<Integer>> AREA_EFFECT_CLOUD_DURATION_ON_USE
+            = KeyFactory.fake("AREA_EFFECT_CLOUD_DURATION_ON_USE");
```

Comment:
We typically don't put those on newlines anyway since its not like you ever read the dummy initialization. Just keep them in one line, the line length doesn't matter.